styleguide/_themes/derek/scss/snowflakes/events.scss

```diff
@@ -0,0 +1,364 @@
+  font-size: 2.2rem;
+  margin-bottom: .75rem;
+  margin-top: 2.25rem;
+  padding: 1.5rem;
+  text-align: center;
+  text-transform: uppercase;
+
+  &:first-child {
+    margin-top: 0;
+  }
+}
+
+// Individual event page.
+.event {
+  @include make-container;
+}
+
+.event-backToEvents {
+  color: $teal-dark;
+  display: block;
+  margin-bottom: 1rem;
+  margin-left: $grid-gutter-width / 2;
+
+  &::before {
+    @include border-radius(50%);
+    background: $teal-dark;
+    border: 0;
+    color: $white;
+    content: '<';
+    margin-right: .5rem;
+    padding: 0 .5rem;
+    text-decoration: none;
+  }
+
+  &:hover {
+    color: $gray-darkest;
+
+    &::before {
+      background: $gray-darkest;
+      color: $white;
+      text-decoration: none;
+    }
+  }
+}
+
+.event-date {
+  @include make-sm-column(2);
+  margin-bottom: 1rem;
+}
+
+.event-dateBox {
+  background: $teal-dark;
+  color: $white;
+  font-family: $subhead;
+  font-size: 2.2rem;
+  font-weight: 400;
+  line-height: 1.5em;
+  margin: 1rem auto;
+  max-width: 100px;
+  padding: 1rem 0;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.event-dateBox-day {
+  font-size: 2em;
+}
+
+.event-details {
+  @include make-sm-column(6);
+}
+
+.event-detailsTitle,
+.event-info-whereTitle,
+.event-info-whenTitle {
+  font-weight: 600;
+  text-transform: uppercase;
+}
+
+.event-info {
+  @include make-sm-column(10);
+  margin-bottom: 1rem;
+}
+
+.event-info-header {
+  @include make-row;
+  margin-bottom: .5rem;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.event-info-headerTitle {
+  @include make-xs-column(12);
+  @include make-md-column(9);
+  font-size: 2.5rem;
+  line-height: 1.5em;
+  margin-bottom: .5rem;
+  padding: $no-padding;
+}
+
+.event-info-headerType {
+  @include make-xs-column(12);
+  @include make-md-column(3);
+  font-weight: 600;
+  margin-bottom: .5rem;
+  padding: $no-padding;
+  text-transform: uppercase;
+}
+
+.event-info-whereWhen {
+  @include make-row;
+  border-top: 2px solid $gray-light;
+  padding-top: 1rem;
+}
+
+.event-info-where {
+  @include make-sm-column(5);
+  margin-bottom: 1rem;
+}
+
+.event-info-when {
+  @include make-sm-column(5);
+  margin-bottom: 1rem;
+}
+
+.event-map {
+  @include make-sm-column(6);
+  margin-bottom: 2rem;
+}
+
+.event-row {
+  @include make-row;
+}
+
+
+// Responsive behavior.
+@media (min-width: $screen-sm-min) {
+  .event,
+  .eventsOverview {
+    padding: $half-padding;
+  }
+
+  .eventsOverview-searchFilter,
+  .eventsOverview-searchResults {
+    margin-bottom: 0;
+  }
+
+  .event-backToEvents {
+    margin-bottom: 0;
+    margin-left: 0;
+  }
+
+  .event-dateBox {
+    margin: 0 0 0 (-$grid-gutter-width / 2);
+  }
+
+  .event-info-headerTitle {
+    text-align: left;
+  }
+
+  .event-info-headerType {
+    text-align: left;
+  }
+
+  .event-info-where {
+    padding: $no-padding;
+  }
+
+  .event-info-when {
+    padding-left: .6rem;
+  }
+
+  .event-map {
+    padding-left: $no-side-padding-left;
```

Comment:
I honestly didn't either, until PhpStorm auto-filled that in.  I don't love it, and would just assume set `0`.